node/network/bridge/src/network.rs

```diff
@@ -180,4 +188,24 @@ impl Network for Arc<sc_network::NetworkService<Block, Hash>> {
 
 		Box::pin(ActionSink(&**self))
 	}
+
+	fn start_request(&self, req: Requests) {
+		let (
+			protocol,
+			OutgoingRequest {
+				peer,
+				payload,
+				pending_response,
+			},
+		) = req.encode_request();
+
+		// Use the equally named method as implemented in `NetworkService`:
+		self.as_ref().start_request(
```

Comment:
Fully qualified syntax might be preferable in this case, for clarity which would also obsolete the comment.